components/provisioner/internal/installation/resource_cleanup.go

```diff
@@ -231,6 +232,24 @@ func (s *serviceCatalogClient) deleteServiceInstances(serviceInstances []v1beta1
 	}
 }
 
+func (s *serviceCatalogClient) removeClusterBrokersFinalizers(brokers []v1beta1.ClusterServiceBroker) {
+	for _, broker := range brokers {
+		logrus.Debugf("trying to delete ClusterServiceBroker %q", broker.Name)
```

Comment:
this log line might be confusing, because we do not try to delete `ClusterServiceBrokers` here, just clearing their finalizers.